css/main.css

```diff
@@ -897,6 +897,27 @@ body.blog {
 .smaller-font {
     font-size: 0.8em;
 }
+
 .github-star-button {
     float: right;
```

Comment:
There's a closing brace missing here. None of the images work properly because of it. For instance /releases#latest shows huge images.